src/python/WMCore/Algorithms/MiscAlgos.py

```diff
@@ -35,7 +35,7 @@ def sortListByKey(data, key):
                 logging.error("Found list entry with empty key set in sortListByKey: %s" % entry)
                 logging.error("Skipping")
                 continue
-        if not value in final.keys():
+        if not value in final:
```

Comment:
```suggestion
        if value not in final:
```

Reply:
Why do I keep not noticing this? it's like the 4th time you suggest this change to me! Thanks!